fix param key of `update uefi` request body

hanasuke opened the pull request:
resolve #79 

I fixed request body of `Update UEFI` for baremetal server.

before:
```
$ ecl --debug baremetal uefi update XXX --settings '{"SataSecureErase":{"value":"Enabled"}}'
START with options: ['--debug', 'baremetal', 'uefi', 'update', 'XXX', '--settings', '{"SataSecureErase":{"value":"Enabled"}}']
: (snip)
REQ: curl -g -i -X PUT https://baremetal-server-jp1-ecl.api.ntt.com/v2/tenant_id/servers/XXX/uefi -H "Accept: application/json" -H "X-Auth-Token: {SHA1}XXX" -H "User-Agent: python-bearclient" -H "Content-Type: application/json" -d '{"uefi": {"setting": {}, "settings": {"SataSecureErase": {"value": "Enabled"}}}}'
```

after:
```
$ ecl --debug baremetal uefi update XXX --settings '{"SataSecureErase":{"value":"Enabled"}}'
START with options: ['--debug', 'baremetal', 'uefi', 'update', 'XXX', '--settings', '{"SataSecureErase":{"value":"Enabled"}}']
: (snip)
REQ: curl -g -i -X PUT https://baremetal-server-jp1-ecl.api.ntt.com/v2/tenant_id/servers/XXX/uefi -H "Accept: application/json" -H "X-Auth-Token: {SHA1}XXX" -H "User-Agent: python-bearclient" -H "Content-Type: application/json" -d '{"uefi": {"setting": {"SataSecureErase": {"value": "Enabled"}}}}'
```

hanasuke committed:
fix param key of `update uefi` request body

kazyshr commented on eclcli/bare/v2/uefi.py:
```diff
@@ -70,7 +70,7 @@ def take_action(self, parsed_args):
 
         body = {"uefi": {"setting": {}}}
         if parsed_args.settings:
```
```suggestion
        if parsed_args.setting:
```

hanasuke replied:
fixed

kazyshr commented on eclcli/bare/v2/uefi.py:
```diff
@@ -70,7 +70,7 @@ def take_action(self, parsed_args):
         body = {"uefi": {"setting": {}}}
         if parsed_args.settings:
-            body['uefi']['settings'] = json.loads(parsed_args.settings)
+            body['uefi']['setting'] = json.loads(parsed_args.settings)
```
```suggestion
            body['uefi']['setting'] = json.loads(parsed_args.setting)
```

hanasuke replied:
fixed

kazyshr reviewed (changes requested):
Thanks for the PR.
Can you rename all "settings" to "setting" over UpdateUEFI class including comments?

hanasuke committed:
add alias '--setting' to '--settings' on  for baremetalserver

hanasuke commented:
I renamed `parsed_args.settings` to `parsed_args.setting`.

I also added the `--setting` alias to the argument `--settings` .
`--settings` is maintained for consistency with official documentation and compatibility with older scripts.
ref: https://ecl.ntt.com/documents/tutorials/eclc/rsts/bare/uefi/update.html#id3

hanasuke commented:
This is execution logs.
* `ecl --debug baremetal uefi update server_id --settings '{"SataSecureErase":{"value":"Enabled"}}'`
```
: snip
REQ: curl -g -i -X PUT https://baremetal-server-jp1-ecl.api.ntt.com/v2/tenant_id/servers/server_id/uefi -H "Accept: application/json" -H "X-Auth-Token: {SHA1}XXX" -H "User-Agent: python-bearclient" -H "Content-Type: application/json" -d '{"uefi": {"setting": {"SataSecureErase": {"value": "Enabled"}}}}'
```
* `ecl --debug baremetal uefi update server_id --setting '{"SataSecureErase":{"value":"Disabled"}}'`
```
REQ: curl -g -i -X PUT https://baremetal-server-jp1-ecl.api.ntt.com/v2/tenant_id/servers/server_id/uefi -H "Accept: application/json" -H "X-Auth-Token: {SHA1}XXX" -H "User-Agent: python-bearclient" -H "Content-Type: application/json" -d '{"uefi": {"setting": {"SataSecureErase": {"value": "Disabled"}}}}'

```

kazyshr reviewed (approved):
LGTM 👍
